src/screens/TrackingFlights/TrackingFlights.jsx

```diff
@@ -1,9 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import {
+  getItemFromLocalStorage,
+  saveItemInLocalStorage,
+} from 'utils/localStorage';
+import Offer from 'shared/components/Offer/Offer';
+import DeleteButton from 'shared/components/DeleteButton/DeleteButton';
 
 import './styles.scss';
 
 function TrackingFlights() {
-  return <div>tracking </div>;
-}
+  const [quotes, setQuotes] = useState([]);
+  const [qutoeDelete, setQuoteDelete] = useState(0);
+
+  useEffect(() => {
+    const quotes = getItemFromLocalStorage('quotes');
+    setQuotes(quotes);
+  }, []);
 
+  function deleteOffer(price) {
+    setQuoteDelete(price);
+    const quoteDelete = quotes.filter((elem) => elem.price !== price);
+    const quotesWithoutDeletedQuote = quoteDelete.map((elem) => elem);
+    setQuotes(quotesWithoutDeletedQuote);
+    saveItemInLocalStorage('quotes', quotesWithoutDeletedQuote);
+  }
```

Comment:
This should be an action, and the logic should be kept in reducer. You shouldn't filter by `price`. Each item should have an unique id. What if there will be few connections with the same price?

Reply:
Fixed